app-backend/database/src/main/scala/com/azavea/rf/database/tables/ScenesToProjects.scala

```diff
@@ -111,8 +111,31 @@ object ScenesToProjects extends TableQuery(tag => new ScenesToProjects(tag)) wit
       )
     }
   }
+  
+  def setColorCorrectParamsBatch(projectId: UUID, params: BatchParams)(implicit database: DB) = {
+    val results = for {
```

Comment:
Avoiding inconsistent state is the entire point of doing a batch update - the fact that we save on the number of requests is incidental in this case. Chris has reminded me that the operation does not need to be an upsert, since we set a default color correction when the project_to_scene row is created - we should be able to just do a batch update, and determine if the correct number have been updated.